contrib/coreos/user-data.example

```diff
@@ -176,3 +176,10 @@ write_files:
     content: |
       [Coredump]
       Storage=none
+  - path: /run/deis/bin/docker-cleanup
+    permissions: '0755'
+    content: |
+      #!/bin/bash
+      docker rm `docker ps -a --no-trunc | grep '\sExited\s' | grep -v '\sdeis-.*$' | awk '{print $1 }'`
+      docker run -v /var/run/docker.sock:/var/run/docker.sock -v /var/lib/docker:/var/lib/docker --rm martin/docker-cleanup-volumes:1.5.0
```

Comment:
martin/docker-cleanup-volumes:1.5.0 sourcecode please and it might as well be a deis repo for trust